test/Microsoft.ApplicationInsights.AspNet.Tests/Helpers/HttpContextAccessorHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Framework.DependencyInjection;
+using System;
+
+namespace Microsoft.ApplicationInsights.AspNet.Tests.Helpers
+{
+    public static class HttpContextAccessorHelper
+    {
+        public static HttpContextAccessor CreateHttpContextAccessor(RequestTelemetry requestTelemetry = null, ActionContext actionContext = null)
```

Comment:
It Is fine that it is hidden. This method will be used in 90% of tests and it gives you predefined container. You are not really interested in how it is created.